nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -378,3 +393,44 @@ def scale_insertion_command(source_node, next_nodes, source_node_output_port, sc
         return OVCommandCreator.multiply_insertion_command(
             source_node, next_nodes, source_node_output_port, scale, f"{source_node.node_name}/awq_mul"
         )
+
+
+class OVMixedPrecisionAlgoBackend(OVWeightCompressionAlgoBackend):
```

Comment:
I would suggest to introduce abstract class for `MixedPrecisionAlgoBackend`.

Reply:
Done